src/main/kotlin/calculator/domain/CustomReadStrategy.kt

```diff
@@ -0,0 +1,22 @@
+package calculator.domain
+
+import calculator.interfaces.StringReadStrategy
+
+class CustomReadStrategy() : StringReadStrategy {
+    override fun readString(string: String): List<String> {
+        val separator = getCustomSeparator(string)
+        val stringWithOutCustomSeparator = getStringWithOutCustomSeparator(string)
+
+        return stringWithOutCustomSeparator.split(separator!!)
+    }
+
+    private fun getCustomSeparator(string: String): String? {
+        val regexResult = Regex("//(.)\\\\n(.*)").find(string)
```

Comment:
수많은 이스케이프가 필요한 문자열의 경우 3중 따옴표 문자열을 사용하면 더 깔끔하게 표현할 수 있습니다.

```suggestion
        val regexResult = Regex("""//(.)\\n(.*)""").find(string)
```